kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -977,8 +983,20 @@ public LocalDate retrieve(String value) {
 
         return mapping;
     }
+    public void removeSpecialChar(ArrayList<String > listOfIds,Map<String, Object> record){
```

Comment:
Please move this method to some util class, and not a service class

Reply:
> Please move this method to some util class, and not a service class. Also, create a string constant for the regular expression instead of hardcoding here in the logic.

there is a method which validate referance date that's why I thought  this is a validation on ids
So i put it here